e2e/protractor.conf.js

```diff
@@ -0,0 +1,26 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+const { SpecReporter } = require('jasmine-spec-reporter');
+exports.config = {
+    allScriptsTimeout: 11000,
+    specs: [
+        './src/test/*.e2e-spec.ts'
+    ],
+    capabilities: {
+        'browserName': 'chrome'
+    },
+    directConnect: true,
+    baseUrl: 'http://localhost:4200/',
```

Comment:
We will need to add more configurations here for testing dynamic environments for pipeline.